dts/bindings/led_strip/worldsemi,ws2812-rpi_pico-pio.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2023, TOKITA Hiroshi
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Worldsemi WS2812 LED strip, RaspberryPi Pico's PIO binding
+
+  Driver bindings for controlling a WS2812 or compatible LED
+  strip with a PIO feature.
+
+  Parent PIO device's configure the pin that use for output signals.
+
+compatible: "worldsemi,ws2812-rpi_pico-pio"
+
+include: [base.yaml, ws2812.yaml, "raspberrypi,pico-pio-device.yaml"]
```

Comment:
Hi @mbolivar-nordic. Having different bindings for a same compatible based on the bus type is a nice feature that I was not aware of. Yes I agree it would be a bit cleaner. For the I2C and GPIO unified binding how does it work to require a property OR another ?

Reply:
This is a widely effect topic.
I think it is good to discuss in separate topic.